JS-Algorithms/challenges.js

```diff
@@ -70,7 +87,28 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  const modBase = Math.pow(10, lastDigits + 1);
+
+  const serie = Array.from({ length: number }, (_, idx) => idx + 1);
+
+  const poweredSerie = serie.map((num) => {    
+    let poweredNumber = num;
+
+    for (let i = 1; i < num; i++) {
+      poweredNumber *= num;
+      poweredNumber %= modBase;
+    }
+
+    return poweredNumber;
+  });
+
+  const sumOfPowers = poweredSerie
+    .reduce((acc, currNum) => (acc + currNum) % modBase, 0)
```

Comment:
sabe about naming conventions
```suggestion
    .reduce((acc, currentNumber) => (acc + currentNumber) % modBase, 0)
```